src/backend/TrafficCourtsApi/Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
-WORKDIR /app
+# FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
+FROM registry.redhat.io/rhel8/dotnet-50-runtime AS base
+WORKDIR /opt/app-root
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+# FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+FROM registry.redhat.io/rhel8/dotnet-50 AS build
```

Comment:
Shouldn't we be using the ubi8 instead of rhel8? https://developers.redhat.com/products/rhel/ubi    rhel8 has licenced components, ubi8 is free to use.

Reply:
https://developers.redhat.com/articles/ubi-faq#introduction

7. What is the difference between the pre-UBI base images and UBI base images?
As with pre-UBI base images, UBI is derived from Red Hat Enterprise Linux. It differs from pre-UBI Red Hat Enterprise Linux 7  base images in a number of ways, most notably the fact that it can be freely redistributed under the terms of the Red Hat Universal Base Image End User License Agreement (EULA). In Red Hat Enterprise Linux 8, all base images are governed by the UBI EULA. See the UBI documentation section, “How are UBI images different?”

When deploying on OpenShift and/or Red Hat Enterprise Linux, developers can use any package accessible via their Red Hat subscription. When deploying on non-Red Hat platforms, developers can only use the reduced set of packages that are tagged with the UBI-EULA. 

“UBI content” and “non-UBI content” are different primarily due to the EULA. The former can be freely redistributable; the latter is not.
 
8. Will applications built on UBI have access to the same content as images built on a non-UBI base image?
Yes, but accessing non-UBI content requires a Red Hat subscription and renders containers built with non-UBI content non-redistributable.